query/src/sessions/query_ctx.rs

```diff
@@ -221,6 +221,12 @@ impl QueryContext {
     /// SELECT * FROM (SELECT * FROM db.table_name) as subquery_1, (SELECT * FROM db.table_name) AS subquery_2
     /// ```
     pub async fn get_table(&self, database: &str, table: &str) -> Result<Arc<dyn Table>> {
+        if database.eq_ignore_ascii_case("INFORMATION_SCHEMA") {
```

Comment:
With this ,so we don;t need `impl<const UPPER: bool> ColumnsTable<UPPER>`

Reply:
You are right!

Reply:
It would be better to add a map `case_insensitive_databases` to mark these database's table is case insensitive?

Reply:
How about keep these then we can have upper & lower case db/tables in `select * from system.tables` ?

Reply:
I think `INFORMATION_SCHEMA ` main for MySQL driver/BI usage, `select * from system.tables` no related to it?

Reply:
We can keep only one like `INFORMATION_SCHEMA` and its tables with upper-case too?